### PR TITLE
testing/h1-cli: Upgrade to 1.6.1

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

File: testing/h1-cli/APKBUILD [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
 _pkgname=${pkgname/-cli/}
-pkgver=1.6.0
+pkgver=1.6.1
 pkgrel=0
 pkgdesc="Command-line interface for HyperOne platform"
 url="https://www.hyperone.com/tools/cli/"
 arch="x86_64"
 license="MIT"
-depends="openssh-client"
+depends="openssh-client libstdc++"
```
Comment (nmeum) [inline testing/h1-cli/APKBUILD]: This change shouldn't be necessary `g++` depends on `libstdc++` and `build-base` depends on `g++`.

**Comment by ad-m** [comment]
@nmeum , fixed!

**Comment by maxice8** [comment]
Please squash everything into a single commit

**Comment by ad-m** [comment]
@maxice8 , done!

**Pull request closed without merging**